draft-ietf-taps-interface.md

```diff
@@ -889,6 +892,20 @@ by using the following call on the Connection Object:
 TransportProperties := Connection.GetTransportProperties()
 ~~~
 
+An individual property is then queried via the following call on the
+`TransportProperties` object:
+
+~~~
+value := TransportProperties.Get(property)
+~~~
+
+If the given Transport Property is of type Boolean, `Has` may be provided as an
+alias for `Get`, useful for readability in such constructions as the following:
+
+~~~
+if TransportProperties.Has(boolean_property) then ...
```

Comment:
I'd suggest removing get and has here, and just specifying a bit more detail on Connection.GetProperties()